src/main/java/com/nvs/store/controllers/CartController.java

```diff
@@ -1,4 +1,49 @@
 package com.nvs.store.controllers;
 
+
+import com.nvs.store.common.ApiResponse;
+import com.nvs.store.dto.cart.AddToCartDto;
+import com.nvs.store.dto.cart.CartDto;
+import com.nvs.store.service.CartService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/api/v1/auth/cart")
```

Comment:
1. please remove auth from path - there is no sense in it => "/api/v1/cart"
2. please use REST naming convention in mappings ( [(https://restfulapi.net/resource-naming/)] )
Cart actions
-  GET - "/carts"
-  POST - "/carts"
-  PUT - "/carts/{cartId}"
-  DELETE - "/carts/{cartId}"